packages/analyzer/src/core/analysis-context.ts

```diff
@@ -71,6 +71,7 @@ export class AnalysisContext {
   readonly parsers = new Map<string, Parser<ParsedDocument>>([
     ['html', new HtmlParser()],
     ['js', new JavaScriptParser()],
+    ['mjs', new JavaScriptParser()],
```

Comment:
Of course, you could create a single `const jsParser = new JavaScriptParser();` and then use the the same instance for both `js` and `mjs`.

Reply:
You can also skip all the `import`/`export` scanning for `.mjs` files.

Reply:
Yeah, we want to add more contextual inference for treating a file as a script vs module, and .mjs would be a strong signal.

No objection to landing this as is though.